app/styles/form-elements/_checkbox.scss

```diff
@@ -416,6 +416,9 @@ $checkButtonBorderRadius: $border-radius__base--small;
     &.sd-check-button__group--compact {
         gap: $sd-base-increment * 0.5;
     }
+    &.sd-check-button__group--no-wrap {
```

Comment:
does that classname that you're styling exist on some HTML element? I couldn't find it

Reply:
Yep but on client-core side